deps.edn

```diff
@@ -3,7 +3,7 @@
            aysylu/loom                       {:mvn/version "1.0.2"}
            camel-snake-kebab                 {:mvn/version "0.4.2"}
            edu.stanford.nlp/stanford-corenlp {:mvn/version "4.2.0"}
-           techascent/tech.ml                {:mvn/version "5.01"}
+           scicloj/metamorph.ml              {:mvn/version "0.3.0-alpha3"}
```

Comment:
There is a transition from `tech.ml` ongoing to:
https://github.com/scicloj/metamorph.ml - branch mergeTechML 
https://github.com/scicloj/scicloj.ml  (which is a shim without code, just re-organizing namespaces from various projects)

The "feature" of documenting options as part of model metadata is not in `tech.ml`, and will likely not go there ever.
If all goes well, `tech.ml` will be feature frozen and deprecated soon. 

So we need it "successor" here , namely `methamorph.ml`

Reply:
My apologies, but I don't think I quite get the relations between the different projects... metamorph.ml is succeeding tech.ml, but the idea is to merge the code into scicloj.ml in the future, so metamorph.ml is just another step on the way towards a scicloj.ml? But why the sudden deprecation of tech.ml?